### DISCUSSION TITLE
Can you advertise as the peripheral?

### DISCUSSION THREAD

**Discussion by eraguzin** [JuulLabs/kable#524] [Q&A] [answered]
I've been going through the examples and see that you can scan as the central to connect. Does the library allow you advertise as a peripheral to be connected to? Is that within the scope of future development of the library?

**Comment by twyatt** [comment, marked as answer]
This was requested in #51, so I'd subscribe to that issue for updates.

Although this feature is likely a long ways away (if ever) — as I don't foresee myself having the time to take this on.
If the community has strong interest in this, then I'm happy to have design discussions in the hopes of outside contributions/pull-requests.
